### PR TITLE
Add permissions matrices to the Actions workflows

### PR DESCRIPTION
**Pull request by JamesMGreene**
Because `#bestpractices`, and to get these `permissions` matrices in place before they become a hard requirement in the future. 🛡️

### CODE REVIEW

**Review by preechapon-cmd — APPROVED** [review]

File: .github/workflows/2-commit-a-file.yml
Comment (dragonthundersexgod-maker) [inline .github/workflows/2-commit-a-file.yml:1]: Null

**Review by dragonthundersexgod-maker — COMMENTED** [review]
🤩- [ ] []()~~````~~

**Comment by dragonthundersexgod-maker** [comment]
> Because `#bestpractices`, and to get these `permissions` matrices in place before they become a hard requirement in the future. 🛡️ 😵

**Review by RoosterRoll — COMMENTED** [review]
Ok

**Pull request merged**